addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
@juwara0 Thanks for the explanation, that make more sense to me than commenting one out temporarily did. I've actually liked doing things one-per-line like that in the past for exactly that same reason, it makes the diffs look nicer too. However, since we can't have trailing commas on everything that's a list of things (in some cases it's OK, although not with our lint rules) and in some cases I think even js doesn't allow it) I've gotten used to the fact that I'm gonna have to edit lists of things that are in-line and not broken out. I don't think I'd want all my function parameters, for instance one per line, or all the dependencies of my computed property listed out one per line. I think that by breaking out the named imports one per line all the time, we're being inconsistent in that regard. If we're gonna do that, for all the reasons you mentioned, we should probably do it across the board, don't you think, with all lists of things?